x-pack/plugins/security_solution/public/detections/components/host_isolation/helpers.ts

```diff
@@ -18,7 +18,18 @@ export const getFieldValues = (
   },
   data: TimelineEventsDetailsItem[] | null
 ) => {
-  return find({ category, field }, data)?.values;
+  const categoryCompat =
+    category === 'signal' ? 'kibana' : category === 'kibana' ? 'signal' : category;
+  const fieldCompat =
+    category === 'signal'
+      ? field.replace('signal', 'kibana.alert').replace('rule.id', 'rule.uuid')
+      : category === 'kibana'
+      ? field.replace('kibana.alert', 'signal').replace('rule.uuid', 'rule.id')
+      : field;
+  return (
```

Comment:
I think it's coming together now. We keep both versions and search for each if we can't find one or the other. very interesting